### DISCUSSION TITLE
Where is the "Show examples on Github"  suggestion?

### DISCUSSION THREAD

**Discussion by SteveW555** [community/community#127073] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

**Show Examples on Github links**
I used to see this often, but I can't remember what mode/panel/window of Copilot was showing it.
I'm using Copilot in Visual Studio 2022, and since upgrading to the new version of Copilot for VS 17.10, I can't find it anymore.
I tried asking Copilot chat, but just got the "As an AI... reply"
Thanks

**Comment by ayim** [comment, marked as answer]
If "Enable API Usage Examples" is still enabled, you should see the link to GitHub Examples when you mouseover a method.
![image](https://github.com/community/community/assets/10951869/5f283dcd-cc84-4a5c-bb00-6caaef25d2ce)
![image](https://github.com/community/community/assets/10951869/f14dde71-b929-4e36-9367-ad825f49a44b)

Not all functions are supported, functions from publicly published libraries are generally a good bet.
If you're unable to get API Usage Examples upon mouseovering a variety of methods, please Report a Problem in Visual Studio.

![image](https://github.com/community/community/assets/10951869/2f5bc896-cbdd-4c74-a15e-58d4dfb1fd59)

**Comment by SteveW555** [comment]
Thank you very much for this, I see it now.
